examples/app-npm-webpack/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "globalize-full-app-npm-webpack",
```

Comment:
Dunno how we generally do that, but if a package is marked `private` giving a `name` is kinda moot.